File: project3/Team2/backend/src/main/java/bitc/full502/backend/dto/AgencyProductDTO.java
package bitc.full502.backend.dto;

import lombok.Getter;
import lombok.Setter;
import java.time.LocalDate;

@Getter
@Setter
public class AgencyProductDTO {
    private int pdKey;
    private String agName;
    private String pdNum;
    private String pdProducts;
    private int pdPrice;
    private LocalDate apStore;
    private int stock;
}

File: project3/Team2/android/app/src/main/java/bitc/fullstack502/finalproject/AgencyProductDTO.kt
package bitc.fullstack502.finalproject

data class AgencyProductDTO(
    val agName: String,
    val pdNum: String,
    val pdProducts: String,
    val apPrice: Int,
    var quantity: Int = 0,        // 선택 수량
    var isSelected: Boolean = false
)
